test/controllers/sessions_controller_test.rb

```diff
@@ -1,5 +1,30 @@
 require "test_helper"
 
 describe SessionsController do
+  describe "login" do
+    it "can login" do
+      user = User.first
+      login(user)
 
+      user_id = session[:user_id]
+
+      user_id.must_equal user.id
+    end
+  end
+
+  describe "logout" do
+    it "ends the session" do
+      user = User.first
+      login(user)
+
+      user_id = session[:user_id]
+
+      user_id.must_equal user.id
+
+      post logout_path, params: {user_id: user_id}
+      user_id = session[:user_id]
```

Comment:
Checking the session in this test is fine - but I'd like to see you also testing the controller as far as far what the response status is and where it goes after this request completes